filament/backend/include/private/backend/CommandStream.h

```diff
@@ -205,7 +205,11 @@ class NoopCommand : public CommandBase {
 
 // ------------------------------------------------------------------------------------------------
 
-#ifdef NDEBUG
+#if defined(_MSC_VER)
+    // official syntax is "#define identifier( params, ... )" and MSVC chokes on "params..."
+    // see: https://en.cppreference.com/w/cpp/preprocessor/replace
+    #define DEBUG_COMMAND(methodName, params, ...)
```

Comment:
I think it's fine to use this syntax everywhere (we already do in other parts of our codebase), so no `#if` check needed.

Reply:
@bejado, I made the change